test/api/controllers/routerController/executeFromRest.test.js

```diff
@@ -137,14 +138,63 @@ describe('Test: routerController.executeFromRest', function () {
       catch (e) {
         done(e);
       }
-    }, 10);
+    }, 5);
 
-    setTimeout(function () {
-      if (timer === false) {
+    setTimeout(() => {
```

Comment:
This can be done more simply, by using the should method "this.timeout(<timeout in ms>)"

If the done() callback is not called before the timeout is reached, the test fails